c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -29,6 +39,7 @@ INSTANCE_ID =
 endif
 
 PIP_CMD ?= $(VENV_BIN)/pip
+PIP_INSTALL_ARGS += install --trusted-host pypi.camptocamp.net
```

Comment:
Ah.... but I need the `--trusted-host` otherwise pip throws a SSL error...

What are we then using now ?